R/calc_match_person_db.R

```diff
@@ -275,7 +275,7 @@ calc_match_person_db <- function(df_one, id_one, forename_one, surname_one, dob_
 
     # Determine missing non-match fields
     non_matches <- df_one %>%
-      dplyr::anti_join(y = matches %>% dplyr::select(ID_ONE)) %>%
+      dplyr::anti_join(y = matches %>% dplyr::select(ID_ONE), by = "ID_ONE") %>%
       dplyr::mutate(
```

Comment:
The anti-join did not previously explicitly state to join by "ID_ONE" so at run-time was presenting a warning message simply stating this is what was happening.

Defining the join prevents the message.